blocks/results-list-block/features/results-list/results/index.jsx

```diff
@@ -21,6 +23,8 @@ const Results = ({
 	showImage = false,
 	showItemOverline = false,
 	targetFallbackImage,
+	overline,
+	overlineURL,
 }) => {
 	const [queryOffset, setQueryOffset] = useState(configuredOffset);
 
```

Comment:
Unable to comment lower - but we need to update the `filter` of the `useContent` call to remove the `resized_params` and add in `auth`